cli/sawtooth_cli/main.py

```diff
@@ -83,6 +84,18 @@ def create_parent_parser(prog_name):
         action='count',
         help='enable more verbose output')
 
+    try:
+        version = pkg_resources.get_distribution('sawtooth-cli').version
+    except pkg_resources.DistributionNotFound:
+        print("please install sawtooth-cli with setup.py")
+        return
+
+    parent_parser.add_argument(
+        '-V', '--version',
+        action='version',
+        version='%(prog)s {}'.format(version),
```

Comment:
Let's do '%(prog)s (Hyperledger Sawtooth) {}'.  This is somewhat similar to some GNU utilities like grep, sed, etc.

I may be better to do 'sawtooth-cli (Hyperledger Sawtooth) {}' to make it clear that it's the sawtooth-cli's version number explicitly.  This is a bit confusing with the using %(prog)s since it is 'sawtooth' which is ambiguous.